crowbar_framework/app/controllers/api/storages_controller.rb

```diff
@@ -15,6 +15,8 @@
 #
 
 class Api::StoragesController < ApiController
+  before_action :set_storage
```

Comment:
right, will change it here, for the controller name, we have to see if that creates namespacinng problems, as we already have a ceph_controller. I would prefer to test it in another PR